.gitattributes

```diff
@@ -29,4 +29,5 @@ src/backend/distributed/utils/citus_outfuncs.c -citus-style
 src/backend/distributed/utils/citus_read.c -citus-style
 src/backend/distributed/utils/citus_readfuncs_95.c -citus-style
 src/backend/distributed/utils/ruleutils_95.c -citus-style
+src/backend/distributed/utils/ruleutils_96.c -citus-style
```

Comment:
It is, I think you were expecting to see a `citus_readfuncs_96` here, but I removed it.